### DISCUSSION TITLE
Compatible Shaderpacks

### DISCUSSION THREAD

**Discussion by BeansBeefBroccoli** [IrisShaders/Iris#869] [Q&A] [answered]
Have the [Vanilla Plus](https://curseforge.com/minecraft/customization/vanilla-plus-shader) shaders on the Vanilla profile and/or the [AstraLex](https://curseforge.com/minecraft/customization/astralex-shader-bsl-edit) shaders been tested? If not, I will test it, but I might not notice every bug if there are some.

**Comment by NoComment1105** [comment, marked as answer]
A list of supported shaders can now be found [here](https://github.com/IrisShaders/Iris/blob/trunk/docs/supportedshaders.md)
